### PR TITLE
Add simple pareto front plot

### PR DESCRIPTION
**Pull request by zchenry**

### CODE REVIEW

**Comment by zchenry** [comment]
For the following code adapted from [this page](https://optuna.readthedocs.io/en/stable/reference/visualization/generated/optuna.visualization.plot_pareto_front.html#optuna.visualization.plot_pareto_front)

```python
import optuna


def objective(trial):
    x = trial.suggest_float("x", 0, 5)
    y = trial.suggest_float("y", 0, 3)

    v0 = 4 * x ** 2 + 4 * y ** 2
    v1 = (x - 5) ** 2 + (y - 5) ** 2
    return v0, v1


if __name__ == "__main__":
    study = optuna.create_study(
        storage="sqlite:///db-multi.sqlite3",
        directions=["minimize", "minimize"])
    study.optimize(objective, n_trials=50)
```

The resulted plot is as follows
paretofront

There are two problems
- Something is wrong with Axis (also GraphHistory?).
- Lint checking failed although there is no error.

**Comment by c-bata** [comment]
> Lint checking failed although there is no error.

Could you try `npm run fmt` ?

**Comment by zchenry** [comment]
It shows same behavior on axis as GraphHistory.

File: optuna_dashboard/static/components/GraphParetoFront.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+  const plotData: Partial<plotly.PlotData>[] = [
+    {
+      type: "scatter",
+      x: completedTrials.map((t: Trial): number => {
+        return t.values![0]
+      }),
+      y: completedTrials.map((t: Trial): number => {
+        return t.values![1]
+      }),
```
Comment (c-bata) [inline optuna_dashboard/static/components/GraphParetoFront.tsx:82]: In multi-objective optimization, 3 or more objective values might exist. How about put two select-boxes to select which objective values show, on the left of the graph?

**Review by c-bata — CHANGES_REQUESTED** [review]
Hi @zchenry. Overall looks good and easy to read!
I left a review comment.

File: optuna_dashboard/static/components/GraphParetoFront.tsx (hunk 1 above)
Comment (zchenry) [reply optuna_dashboard/static/components/GraphParetoFront.tsx:82]: Thank you for your insightful comment! I will work on it.

**Comment by zchenry** [comment]
Axis is now selectable as shown below.

**Review by c-bata — APPROVED** [review]
LGTM! Thank you for your contribution.

**Pull request merged**